roles/pulp_common/tasks/install_pip.yml

```diff
@@ -12,8 +12,6 @@
         name:
           - selinux-policy-devel
           - policycoreutils
```

Comment:
We should just remove this line, it is a dependency of policycoreutils-python / python3-policycoreutils

Reply:
I thought about that, but then kept the diff minimal. Done now.